website/src/_includes/section-menu/sdks.html

```diff
@@ -30,7 +30,7 @@
   <span class="section-nav-list-title">Python</span>
   <ul class="section-nav-list">
     <li><a href="{{ site.baseurl }}/documentation/sdks/python/">Python SDK overview</a></li>
-    <li><a href="{{ site.baseurl }}/documentation/sdks/pydoc/{{ site.release_latest }}/" target="_blank">Python SDK API reference <img src="{{ site.baseurl }}/images/external-link-icon.png"
+    <li><a href="https://beam.apache.org/releases/pydoc/{{ site.release_latest }}/" target="_blank">Python SDK API reference <img src="{{ site.baseurl }}/images/external-link-icon.png"
```

Comment:
Why not use `{{ site.baseurl }}`?

Reply:
Javadocs and pydocs are now published from a separate repository, and won't be present in a local / staged version of the website. So when you're serving locally, links to javadocs/pydocs will point to the production site rather than 404.

Reply:
Ah, that makes sense.